### ISSUE TITLE
Will it be possible to save the players progress

### ISSUE THREAD

**Issue by ghostt187** [noahbkim/cs2s-plugin-dmrank#1]
Hey nice work btw appreciate it, i wanted to ask if it will be possible to save the players kills and progress using database? MYSQL? thanks..

**Comment by noahbkim** [comment]
Yeah, I'm currently figuring out how I want to support that. Probably will make requests to server with native Steam HTTP service and it'll be up to the user to make a server.
